tensorboard/webapp/metrics/views/main_view/common_selectors_test.ts

```diff
@@ -710,23 +711,20 @@ describe('common selectors', () => {
   });
 
   describe('getRenderableRuns', () => {
-    it('returns all runs associated with experiment', () => {
-      const exp1Result = selectors.factories.getRenderableRuns(['exp1'])(state);
-      expect(exp1Result.length).toEqual(2);
-      expect(exp1Result[0].run).toEqual({...run1, experimentId: 'exp1'});
-      expect(exp1Result[1].run).toEqual({...run2, experimentId: 'exp1'});
-
-      const exp2Result = selectors.factories.getRenderableRuns(['exp2'])(state);
-      expect(exp2Result.length).toEqual(3);
-      expect(exp2Result[0].run).toEqual({...run2, experimentId: 'exp2'});
-      expect(exp2Result[1].run).toEqual({...run3, experimentId: 'exp2'});
-      expect(exp2Result[2].run).toEqual({...run4, experimentId: 'exp2'});
+    it('returns all runs associated with each experiment', () => {
+      state.app_routing!.activeRoute!.routeKind = RouteKind.COMPARE_EXPERIMENT;
```

Comment:
I recognize that this style of mocking the state is potentially controversial. This is the way it has been done in this test file up until now so I'd like to continue the practice until we make a decision around which approach to take in the future.